packages/babel-plugin-transform-jsx-to-htm/index.mjs

```diff
@@ -195,7 +216,30 @@ export default function jsxToHtmBabelPlugin({ types: t }, options = {}) {
 				buffer = bufferBefore;
 
 				state.set('jsxElement', true);
-			}
+      },
+      
+      JSXFragment(path, state) {
```

Comment:
This visitor method is mostly duplicate of the `JSXElement` visitor method, so how about using a function like this for less repetition:

```js
function handler(path, state, isFragment) {
	let quasisBefore = quasis.slice();
	let expressionsBefore = expressions.slice();
	let bufferBefore = buffer;

	buffer = '';
	quasis.length = 0;
	expressions.length = 0;

	if (isFragment) {
		processChildren(path.node, '', true);
	}
	else {
		processNode(path.node, path, true);
	}

	quasis = quasisBefore;
	expressions = expressionsBefore;
	buffer = bufferBefore;

	state.set('jsxElement', true);
}
```

Reply:
Done